CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+##[0.15.5] - released on 2018-03-05
+### Changed
```

Comment:
This should probably be `Fixed`, while it's a change it's really a bugfix more than a change in behavior.

Reply:
fair enough